### DISCUSSION TITLE
Full-Page Component without browser reloading

### DISCUSSION THREAD

**Discussion by f-ricci** [livewire/livewire#5553] [1. Help] [answered]
Is there any way to enable [full page components](https://laravel-livewire.com/docs/2.x/rendering-components#page-components) to load without fully reloading the browser?

What I would like is to navigate between routes that render full page components without having the browser fully reloading everything. The same functionality that is possible with turbolinks.

Is there any way to accomplish this without adding turbolinks and the required adapter library?

**Comment by joshhanley** [comment, marked as answer]
@f-ricci currently the only way to achieve that with Livewire is to use turbo/turbolinks, that's why the adapter library exists 🙂 but in V3 there will be a built in SPA mode!

**Reply by f-ricci** [reply]
Great! I saw the announcement of the Version 3.0 , do you have any news for when it will be available?

**Reply by hebbet** [reply]
https://github.com/livewire/livewire/discussions/5261#discussioncomment-4895440

**Reply by joshhanley** [reply]
@f-ricci not as yet, see the comment linked above 🙂
